__e2e__/__snapshots__/config.test.ts.snap

```diff
@@ -0,0 +1,236 @@
+      \\"options\\": [
+        {
+          \\"name\\": \\"--root [string]\\",
+          \\"description\\": \\"[DEPRECATED - root is discovered automatically] Override the root directory for the android build (which contains the android directory)\\",
+          \\"default\\": \\"\\"
+        },
+        {
+          \\"name\\": \\"--variant [string]\\",
+          \\"description\\": \\"Specify your app's build variant\\",
+          \\"default\\": \\"debug\\"
+        },
+        {
+          \\"name\\": \\"--appFolder [string]\\",
+          \\"description\\": \\"[DEPRECATED – use \\\\\\"platforms.android.appName\\\\\\" in react-native.config.js] Specify a different application folder name for the android source. If not, we assume is \\\\\\"app\\\\\\"\\"
+        },
+        {
+          \\"name\\": \\"--appId [string]\\",
+          \\"description\\": \\"[DEPRECATED – use \\\\\\"platforms.android.appName\\\\\\" in react-native.config.js] Specify an applicationId to launch after build.\\",
+          \\"default\\": \\"\\"
+        },
+        {
+          \\"name\\": \\"--appIdSuffix [string]\\",
+          \\"description\\": \\"Specify an applicationIdSuffix to launch after build.\\",
+          \\"default\\": \\"\\"
+        },
+        {
+          \\"name\\": \\"--main-activity [string]\\",
+          \\"description\\": \\"Name of the activity to start\\",
+          \\"default\\": \\"MainActivity\\"
+        },
+        {
+          \\"name\\": \\"--deviceId [string]\\",
+          \\"description\\": \\"builds your app and starts it on a specific device/simulator with the given device id (listed by running \\\\\\"adb devices\\\\\\" on the command line).\\"
+        },
+        {
+          \\"name\\": \\"--no-packager\\",
+          \\"description\\": \\"Do not launch packager while building\\"
+        },
+        {
+          \\"name\\": \\"--port [number]\\",
+          \\"default\\": 8081
+        },
+        {
+          \\"name\\": \\"--terminal [string]\\",
+          \\"description\\": \\"Launches the Metro Bundler in a new window using the specified terminal path.\\",
+          \\"default\\": \\"vscode\\"
+        },
+        {
+          \\"name\\": \\"--tasks [list]\\",
+          \\"description\\": \\"Run custom Gradle tasks. By default it's \\\\\\"installDebug\\\\\\"\\"
+        },
+        {
+          \\"name\\": \\"--no-jetifier\\",
+          \\"description\\": \\"Do not run \\\\\\"jetifier\\\\\\" – the AndroidX transition tool. By default it runs before Gradle to ease working with libraries that don't support AndroidX yet. See more at: https://www.npmjs.com/package/jetifier.\\",
+          \\"default\\": false
+        }
+      ]
+    }
+  ],
+  \\"assets\\": [],
+  \\"platforms\\": {
+    \\"ios\\": {},
+    \\"android\\": {}
+  },
+  \\"project\\": {
+    \\"ios\\": {
+      \\"sourceDir\\": \\"/private/var/folders/zt/917v0jxx6lg3p_zfh9s_02bm0000gn/T/test_root/TestProject/ios\\",
```

Comment:
these paths are going to be different based on the CI they run in (Circle vs Azure), you'll need to replace them with something like `<<REPLACED_ROOT>>` before taking a snapshot